ext_localconf.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-if (!defined('TYPO3_MODE')) die ('Access denied.');
+defined('TYPO3') || die();
+
+use \TYPO3\CMS\Core\Http\ApplicationType;
```

Comment:
Is this line necessary? I see no usage for it.

Reply:
Switched back to the old "TYPO3_MODE" notation. This makes only sense, if we are on TYPO3  version >= 11

Change will come in  with the next push.